front/src/applications/stdcm/components/StdcmResults/StdcmResults.tsx

```diff
@@ -67,9 +68,12 @@ const StcdmResults = ({
     );
   }, [outputs]);
 
-  const simulationPathSteps = hasSimulationResults
-    ? outputs.results.simulationPathSteps
-    : undefined;
+  const simulationPathSteps = useMemo(() => {
```

Comment:
Easier to understand (and harmonized with `StdcmConfig`)
```suggestion
  const markersInfo = useMemo(() => {
```

Reply:
I will do another PR to rename `simulationPathSteps` by `markersInfo` in `ManageTrainSchedule/Map` too :+1: